client/landing/gutenboarding/components/header/index.tsx

```diff
@@ -109,7 +110,9 @@ const Header: FunctionComponent = () => {
 	const currentDomain = domain ?? freeDomainSuggestion;
 
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
-	const domainElement = (
+	const domainElement = domain ? (
+		domain.domain_name
```

Comment:
Reading code, I somehow expected to have this in `currentDomain` const, but I haven't dived in depth to what's going on here after so many frequent changes. 😁 

Ok like this too tho.

Reply:
I moved `currentDomain` declaration to after this logic. Now feels less confusing for sure.